homework.md

```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a scripting language that enables you to create dynamically updating web content.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Git is a version control software and architecture which is easier, faster and github is a web base git architecture which provide collaboration and feedback easy and fast way over internet.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You have a different copy of the repo and make changes over the copy.
```

Comment:
How about this?
"Create a copy of it to my Github account"

Reply:
Thanks it seems better, I fixed it.